src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: circle, area: " + calculateArea() + " sq. units, "
+                + "radius: " + radius + " units, "
+                + "color: " + getColor();
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw(Figure figure) {

```

Comment:
The 'draw' method should take no parameters and use the instance variables directly instead of expecting a 'Figure' parameter.